pages/index.vue

```diff
@@ -147,25 +179,69 @@ export default defineComponent({
      * @todo 1. Send the request to update the task to the backend server.
      * @todo 2. Update the task in the dom.
      */
-    updateTask(_index, _id) {},
+    async updateTask(_index, _id) {         
```

Comment:
The method should have added some checks to ensure that the new ToDo isn't empty and is not the same as the original one. Accordingly, the necessary toasts should be raised to inform the user.